tests/helm/testdata/goldenfile/logs_otc_daemonset/additional.output.yaml

```diff
@@ -96,149 +97,15 @@ spec:
               protocol: TCP
       initContainers: # ensure the host path is owned by the otel user group
         - name: changeowner
+          # yamllint disable-line rule:line-length
           image: public.ecr.aws/docker/library/busybox:1.36.0
           imagePullPolicy: IfNotPresent
           securityContext:
             capabilities:
-              drop:
-                - ALL
               add:
                 - CAP_CHOWN
-          command:
-            - "sh"
-            - "-c"
-            - |
-              chown -R \
-                0:0 \
-                /var/lib/storage/otc
-          volumeMounts:
-            - mountPath: /var/lib/storage/otc
-              name: file-storage
-      volumes:
-        - configMap:
-            defaultMode: 420
-            items:
-              - key: config.yaml
-                path: config.yaml
-            name: RELEASE-NAME-sumologic-otelcol-logs-collector
-          name: otelcol-config
-        - hostPath:
-            path: /var/log/pods
-            type: ""
-          name: varlogpods
-        - hostPath:
-            path: /var/lib/docker/containers
-            type: ""
-          name: varlibdockercontainers
```

Comment:
What's the reason for removing this? Please describe that in PR's description.

Reply:
I think there's a bug in the goldenfile regeneration method that causes these changes. Fixed.